draft-ietf-rats-corim.md

```diff
@@ -1870,12 +1871,12 @@ The way cryptographic signature validation works depends on the specific Evidenc
 For example, in DICE, a proof of liveness is carried out on the final key in the certificate chain (a.k.a., the alias certificate).
 If this is successful, a suitable certification path is looked up in the Appraisal Context, based on linking information obtained from the DeviceID certificate.
 See Section 9.2.1 of {{DICE.Layer}}.
-If a trusted root certificate is found, the usual X.509 certificate validation is performed.
+If a trusted root certificate is found, X.509 certificate validation is performed.
 
 As a second example, in PSA {{-psa-token}} the verification public key is looked up in the appraisal context using the `ueid` claim found in the PSA claims-set.
 If found, COSE Sign1 verification is performed accordingly.
 
-Regardless of the specific integrity protection method used, the Evidence's integrity MUST be validated successfully.
+Regardless of the specific integrity protection method used, the Evidence's integrity MUST be validated successfully else the Verifier MUST NOT proceed to the next stage.
```

Comment:
Evidence that doesn’t pass signature verification should not be admitted as evidence, but it should be up to policy whether one piece of evidence should spoil the bunch, when it comes to composite attesters.

Reply:
Good point. I will reword that.